buildout.cfg

```diff
@@ -1,12 +1,2 @@
 [buildout]
 extends = plone-5.2.x.cfg
-versions=versions
-
-[versions]
-plone.restapi =
-
-plone.schema = 1.3.0
-plone.dexterity = 2.9.8
```

Comment:
I found that issue #1018 occurs because of `plone.dexterity 2.9.10+`. More specifically after https://github.com/plone/plone.dexterity/pull/137

Strangely, it only occurs in Python 2.

The error occurred in Jenkins, but it did not occur in `plone.restapi` GA, because the `plone.restapi` buildout was pinned to `plone.dexterity 2.9.8`.

After the merge of this PR, the error can happen in GA as well.